data/icons/meson.build

```diff
@@ -0,0 +1,56 @@
+actioniconsdir_path = join_paths(pkgdatadir_path, 'icons', 'hicolor')
+appiconsdir_path = join_paths(datadir_path, 'icons', 'hicolor')
+statusiconsdir_path = join_paths(pkgdatadir_path, 'icons', 'hicolor')
```

Comment:
The third line and the first line, should be `join_paths(datadir_path, 'icons', ...)` , right?

Reply:
I'm based on Makefile.am:

```
actioniconsdir = $(pkgdatadir)/icons/hicolor
appiconsdir = $(datadir)/icons/hicolor
statusiconsdir = $(pkgdatadir)/icons/hicolor
```

Reply:
This seems to be a left over from gnome.
Can you please change both files to use `$(datadir)`?